sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -25,6 +25,23 @@ namespace Azure { namespace Core {
     static constexpr IntervalType WindowsToPosixOffsetSeconds = 11644473600LL;
 
   public:
+    /**
+     * @brief Defines the format applied to the fraction part from any @DateFormat
+     *
+     */
+    enum class TimeFractionFormat
+    {
+      /// Decimals are not included when there are no decimals in the source Datetime and any zeros
+      /// from the right are also removed.
+      DropTrailingZeros,
+
+      /// Decimals are included for any Datetime.
+      AllDigits,
+
+      /// Decimals are removed for any Datetime.
+      Truncate
```

Comment:
What happens if I want both truncate and drop trailing zeros? Also, can I control the number of fractional digits to truncate?